scripts/common/repository.js

```diff
@@ -70,9 +70,11 @@ class MonoRepository {
         }),
       )
       .then((v) =>
-        execa
-          .command('git push origin master --tags', { shell: true })
-          .then(() => v),
+        !dryRelease
+          ? execa
+              .command('git push origin master --tags', { shell: true })
+              .then(() => v)
+          : Promise.resolve(v),
```

Comment:
nitpicking for readability

```suggestion
        dryRelease
          ? Promise.resolve(v)
          : execa
              .command('git push origin master --tags', { shell: true })
              .then(() => v),
```